#! /usr/bin/env python

PACKAGE='aricc_2d_vision'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("low_threshold", double_t, 0, "low threshold", 0, 0, 100)
gen.add("ratio", double_t, 0, "ratio", 0, 0, 20)
gen.add("kernel_size", int_t, 0, "kernel size", 3, 3, 100)
gen.add("L2_gradient", bool_t, 0, "L2 gradient", True);

exit(gen.generate(PACKAGE, "aricc_2d_vision", "EdgeFinder"))
